src/turbostat.c

```diff
@@ -1056,9 +1056,21 @@ static int __attribute__((warn_unused_result)) probe_cpu(void) {
     default:
       do_rapl = 0;
     }
+  } else if (vendor_id[0] == 0x68747541 /* "Auth" */ &&
+             vendor_id[1] == 0x69746e65 /* "enti" */ &&
+             vendor_id[2] == 0x444d4163 /* "cAMD" */ &&
+             family == 0x17) {
```

Comment:
Maybe use a union with char array where byte[12] is zeroed, and a string comparison?